src/MatrixProductOperator.jl

```diff
@@ -66,6 +66,7 @@ function MatrixProductOperator{Open}(arrays; χ = nothing, order = (:l, :r, :i,
 end
 
 function MatrixProductOperator{Closed}(arrays; χ = nothing, order = (:l, :r, :i, :o), meta...)
+    println("MPO")
```

Comment:
Be careful about `println` where they shouldn't be.
```suggestion
    println("MPO")
```

Reply:
Yes sorry.